app/frontend/resources/views/layouts/app.blade.php

```diff
@@ -25,7 +30,7 @@
         <nav class="navbar navbar-expand-md navbar-light navbar-laravel fixed-top">
             <div class="container">
                 <a class="navbar-brand" href="{{ url('/') }}">
-                    <img style="height: 14px" src="{{ cdn_link('assets/logo_horizontal.png') }}" alt="Lolibrary logo">
```

Comment:
This line should be assets; I’ll give you an updated config to use digital ocean spaces in dev

Reply:
👍  I'll revert it back. I was trying to get some assets going on my local.